src/Backend/Modules/ContentBlocks/Layout/Templates/Index.tpl

```diff
@@ -12,12 +12,46 @@
 	</div>
 	{/option:showContentBlocksAdd}
 </div>
-{option:dataGrid}
+
 <div class="dataGridHolder">
-	{$dataGrid}
+	{form:filter}
+	<div class="dataFilter">
+		<table>
+			<tbody>
+			<tr>
+				<td>
+					<div class="options">
+						<p>
+							<label for="title">{$lblTitle|ucfirst}</label>
+							{$txtTitle} {$txtTitleError}
+						</p>
+					</div>
+				</td>
+			</tr>
+			</tbody>
+			<tfoot>
+			<tr>
+				<td colspan="99">
+					<div class="options">
+						<div class="buttonHolder">
+							<input id="search" class="inputButton button mainButton" type="submit" name="search" value="{$lblUpdateFilter|ucfirst}" />
+						</div>
+					</div>
+				</td>
+			</tr>
+			</tfoot>
+		</table>
+	</div>
+	{/form:filter}
+	
+	{option:dataGrid}
+		{$dataGrid}
+	{/option:dataGrid}
+
+	{option:!dataGrid}
```

Comment:
This should be outside the dataGrid holder. The dataGrid holder should only be shown when there is a datagrid.